lib/driver.rb

```diff
@@ -0,0 +1,68 @@
+require_relative 'csv_record'
+
+module RideShare
+  class Driver < CsvRecord
+    attr_reader :name, :vin, :trips
+    attr_accessor :status
+
+    def initialize(id:, name:, vin:, status: :AVAILABLE, trips: nil)
+      super(id)
+      @name = name
+      if vin.length != 17
+        raise ArgumentError, "VIN length is invalid"
+      end
+      @vin = vin
+
+      if [:AVAILABLE, :UNAVAILABLE].include?(status.to_sym)
+        @status = status.to_sym
+      else
+        raise ArgumentError, "Invalid status"
+      end
+
+       @trips = trips || []
+    end
+
+    def add_trip(trip)
+      @trips << trip
+    end
+
+    def average_rating
```

Comment:
Incomplete trips should not be including in the calculations of `average_rating` or `total_revenue`